db/sqlc/migrations/000004_privacy_pairs.up.sql

```diff
@@ -0,0 +1,26 @@
+-- privacy_pairs stores the privacy map pairs for a given session group.
+CREATE TABLE IF NOT EXISTS privacy_pairs (
+    -- The group ID of the session that this privacy pair is associated
+    -- with.
+    group_id BIGINT NOT NULL REFERENCES sessions(id) ON DELETE CASCADE,
+
+    -- The key of the privacy pair.
+    real TEXT NOT NULL,
+
+    -- The value of the privacy pair.
+    pseudo TEXT NOT NULL
+);
+
+-- We'll want easy access to the privacy pairs for a given group ID.
+CREATE INDEX IF NOT EXISTS privacy_pairs_group_id_idx ON privacy_pairs(group_id);
```

Comment:
Given what we learned from how indexes are utilised in a prior PR, I wonder if this index is unnecessary as the other 2 unique indexes below will already be used even, when you're only querying with the `group_id`?

Reply:
nit: it's also a golang built-in function

Reply:
> ndex is unnecessary as the other 2 unique indexes below will already be used even, when you're only querying with the group_id?

im not sure that that is necessarily true. And we do add a query that is "where group_id = x" and so having an index dedicated to it will defs be more efficient. I think sometimes the planner may then fall back to sequential scan.  

However, we only make use of this query (at the moment) at the time of creating an autopilot session. So perhaps optimising for efficiency here isnt necessary. 

So cool, will remove for now - we can always add it back